CreateMessage/handler.ts

```diff
@@ -90,13 +94,10 @@ export const MessagePayloadMiddleware: IRequestMiddleware<
   "IResponseErrorValidation",
   ApiNewMessageWithDefaults
 > = request =>
-  new Promise(resolve =>
-    resolve(
-      ApiNewMessageWithDefaults.decode(request.body).mapLeft(
-        ResponseErrorFromValidationErrors(ApiNewMessageWithDefaults)
-      )
-    )
-  );
+  pipe(
+    TE.fromEither(ApiNewMessageWithDefaults.decode(request.body)),
+    TE.mapLeft(ResponseErrorFromValidationErrors(ApiNewMessageWithDefaults))
+  )();
 
```

Comment:
Not sure what kind of style can fit for these kinds of pipes.
```
pipe(
      request.body,
      ApiNewMessageWithDefaults.decode,
      TE.fromEither,
      TE.mapLeft(ResponseErrorFromValidationErrors(ApiNewMessageWithDefaults))
    )()
```
Maybe this way is more clear of which steps are done in the pipe but it's a personal feel :)

Reply:
I agree, thanks! Done [here](https://github.com/pagopa/io-functions-services/pull/154/commits/5cb0e0b8235af908ae3b06cb6f44111375d8df37)